pulsar-io/core/src/main/java/org/apache/pulsar/io/core/ConnectorContext.java

```diff
@@ -68,7 +69,15 @@
      * @return the namespace this source belongs to
      */
     String getNamespace();
-    
+
+    /**
+     * Get subscription type
+     * @return subscription type
+     */
+    default SubscriptionType getSubscriptionType() {
```

Comment:
isn't it better to add this method in SinkContext instead of ConnectorContext ?

it does not make sense to have this for "Sources" for instance